src/App.tsx

```diff
@@ -101,45 +129,275 @@ class App extends React.Component {
+        </AppBar>          
+      </div>
+      <main> 
       <Grid container spacing={8}>           
           {this.state.movies.map((item, i) => (
             <Grid item key={i} sm={6} md={4} lg={3}>
-            <Card>
-              <CardContent>
-                  <Typography gutterBottom variant="h5" component="h2">
-                    {item.title}
-                  </Typography>
+              <Card className={item.title}>
+                <CardHeader 
+                  title = {item.title}
+                  action = {
+                    <IconButton 
+                      onClick={this.handleMenuClick}
+                      aria-owns={anchorEl ? 'cardMenu' : undefined}
+                      aria-haspopup="true"
+                    >
+                      <MoreVertIcon />
+                    </IconButton>
+                  }
+                />
+                <CardMedia
+                  style={{height: 0, paddingTop: '56.25%'}}
+                  image={MoviePH}
+                  title="img"
+                />
+                <CardContent>
                   <Typography>
                     Year: {item.year}<br />
                     Runtime: {item.runtime}min <br />
                     Genres: {this.joinStr(item.genres)}<br />
                   </Typography>
                 </CardContent>
+                <CardActions>
+                  <Checkbox 
+                    checked={this.state.checkBox}/>
+                </CardActions>
               </Card>
-              </Grid>
+            </Grid>
           ))}
-
         </Grid>
+        <Menu
+          id="cardMenu"
+          anchorEl={anchorEl}
+          open={Boolean(anchorEl)}
+          onClose={() => this.setState({ anchorEl: null })}  
+        >
+        <MenuItem onClick={() => this.setState({deleteDialog: true })}>Delete</MenuItem>
+        <MenuItem>Edit</MenuItem>
+
+      </Menu>      
       </main>
+      <div className="dialogs">
+          <Dialog     
+            open={this.state.formsDialog}
+            aria-labelledby="form-dialog-title"
+            >
+            <DialogTitle id="form-dialog-title">Add Movie</DialogTitle>
+            <DialogContent>
+              <Formik
+                initialValues={{ id: '', year: '', runtime: 0, type: 'Movie', title: '', textSearch: '', roles: [], movieId: '', genres: [], }}
+                onSubmit={(values:Movie , actions) => {
+                  
+                  // todo: add failure notification
+                  // if movie params are not full then give this postfailure message
+                  // this.setState({ postFailureAlert: true });
+
+                  // submits post request of new sample movie to axios
+                  axios.post(cors + heliumApi + 'movies', values)
+                  .then(response => this.setState({ postSuccessAlert: true, formsDialog: false}))
+                  .catch(error => {console.log(error.response)})
+                  }
```

Comment:
Try to separate your logic, data, and display code. 

Ie. This axios call should be a method on the component, or if it will be used in multiple components, it should be a function being imported in this file.

Reply:
created new method submit movie!